planning/autoware_route_handler/CMakeLists.txt

```
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.14)
+project(autoware_route_handler)
+
+find_package(autoware_cmake REQUIRED)
+autoware_package()
+
+ament_auto_add_library(${PROJECT_NAME} SHARED
+  src/route_handler.cpp
+)
+
+if(BUILD_TESTING)
+  ament_add_ros_isolated_gtest(test_autoware_route_handler
+    test/test_route_handler.cpp)
+
```

```suggestion

  ament_target_dependencies(test_autoware_route_handler
    autoware_test_utils
  )

```

This fixes the build error. The build error happened because in the `package.xml` file, this `autoware_test_utils` dependency was turned into a `<test_depend>` from `<depend>`.

---

(Tested on my local machine)

---

My suggestion for future porting operations would be to not change anything in the packages during these moving operations.

It adds additional friction to reviews. And the changes become hard to track. These PRs are already very large, let's focus on just moving these package folders, that's all.